mlflow/pyspark/ml/__init__.py

```diff
@@ -390,10 +395,17 @@ def _get_warning_msg_for_fit_call_with_a_list_of_params(estimator):
 
 def _get_tuning_param_maps(param_search_estimator, uid_to_indexed_name_map):
     tuning_param_maps = []
+
+    def gen_log_key(k):
```

Comment:
```suggestion
    def gen_log_key(param):
```

Can we use `param` to clarify what the argument represents?